src/main/java/com/server/RequestParams.java

```diff
@@ -0,0 +1,46 @@
+package com.server;
+
+import java.util.Hashtable;
+
+public class RequestParams {
+
+    private String path;
+    private String method;
+    private Hashtable<String, String> queryComponent;
+
+    public RequestParams(String path, String method) {
+        this.path = path;
+        this.method = method;
+        this.queryComponent = new Hashtable();
+    }
+
+    public RequestParams(String path, String method, Hashtable queryComponent) {
+        this.path = path;
+        this.method = method;
+        this.queryComponent = queryComponent;
+    }
+
+    public String getPath() {
+        return this.path;
+    }
+
+    public String getMethod() {
+        return this.method;
+    }
+
+    public Hashtable getQueryComponent() {
+        return this.queryComponent;
+    }
+
+    public void setPath(String newPath) {
```

Comment:
I'm wondering if we need these public-facing setters since we're assigning the properties in the constructor. What do you think?

Reply:
Makes sense, if we aren't going to use/need them (which we don't now) we shouldn't have them there. Removed.